### PR TITLE
Added Components, Links, Styling and Rearrange files

### PR DESCRIPTION
**Pull request by kbrandon19**
## Description

The following changes were made:

- Moved images for the Resume Builder (RB) to the main src file 
- Added the following files within the Resume-Builder Component Folder
    - Confirmation.js
    - Error.js
    - resume-builder.js
    - ResumeInput.js

- Created a CSS folder for the resume builder within the main CSS folder. This folder has all the styling for the different components. I wanted to try to mimic how the other CSS files were so I created a sort of main hub for the RB styling called resumebuilder.css and this imports all of the styling for the components and within `../css/index.js` it is imported.
- Added styling for the landing and confirmation components. The styling is not final, it will need to be looked over to pull some preexisting styling from the main app
- Added a link in the Nav components so we can access the RB from there - the path is /resume-builder
- Landing.js name was changed to resume-builder.js to match the path from the link, especially since this will be the first page people will see when the link is clicked.

## How Has This Been Tested?

To test to see components on screen I added a link in the Nav, however, when testing I was met with a 404: This page could not be found. (See attached screenshot) Unsure of what the issue could be.
Screen Shot 2022-08-18 at 15 08 34

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by jeromehardaway — APPROVED** [review]

File: src/assets/css/resume-builder/confirmation.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.container{
```
Comment (andrew-t-james) [inline src/assets/css/resume-builder/confirmation.css:1]: Please format file

File: src/assets/css/resume-builder/landing.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
+.container{
```
Comment (andrew-t-james) [inline src/assets/css/resume-builder/landing.css:1]: +1 formatting

File: src/assets/index.css [hunk 3]
```diff
@@ -13,3 +13,6 @@
 @import 'css/pagination.css';
 @import 'css/toggle.css';
 @import 'css/video.css';
+
+/* Styling for Resume Builder */
```
Comment (andrew-t-james) [inline src/assets/index.css:17]: Please match project conventions and co locate similarly

File: src/components/Resume-Builder/Confirmation.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```
Comment (andrew-t-james) [inline src/components/Resume-Builder/Confirmation.js:1]: File banding should use canoe case

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: src/assets/index.css (hunk 3 above)
Comment (kbrandon19) [reply src/assets/index.css:17]: To resolve this I merged all of the styling for each component in a single stylesheet named resumebuilder.css

Screen Shot 2022-08-19 at 22 28 01

File: src/components/ResumeBuilder/Confirmation.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import CheckCircle from '../../images/check-circle.png'
+import '../../assets/css/resumebuilder.css'
```
Comment (andrew-t-james) [inline src/components/ResumeBuilder/Confirmation.js:4]: [This](https://github.com/Vets-Who-Code/vets-who-code-app/pull/421/files#diff-685933f525b831374b30df1f8a8bd36f88ba49e5b34bcbb64203eaaceedea5c1R16) is global. I think these import statements could be dropped.

File: src/components/ResumeBuilder/Error.js [hunk 6]
```diff
@@ -1,5 +1,7 @@
 import React from 'react'
 import { useRouter } from 'next/router'
+import '../../assets/css/resumebuilder.css'
```
Comment (andrew-t-james) [inline src/components/ResumeBuilder/Error.js:3]: +1 on import statement

File: src/components/ResumeBuilder/resume-builder.js [hunk 7]
```diff
@@ -1,6 +1,9 @@
+/* NOTE: This file name was renamed to resume-builder to make the path name 
+read better when linked from the Nav component */
+
 import React from 'react'
 import { useRouter } from 'next/router'
-import '../css/LandingPage.css'
+import '../../assets/css/resumebuilder.css'
```
Comment (andrew-t-james) [inline src/components/ResumeBuilder/resume-builder.js:6]: +1 on the import

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: src/components/ResumeBuilder/Error.js (hunk 6 above)
Comment (kbrandon19) [reply src/components/ResumeBuilder/Error.js:3]: I'm not sure what you mean by, '+1 on import statement'

File: src/components/ResumeBuilder/resume-builder.js (hunk 7 above)
Comment (andrew-t-james) [reply src/components/ResumeBuilder/resume-builder.js:6]: The import is not needed.

File: src/components/ResumeBuilder/Error.js (hunk 6 above)
Comment (andrew-t-james) [reply src/components/ResumeBuilder/Error.js:3]: The import is not needed

Comment (andrew-t-james) [reply src/components/ResumeBuilder/Error.js:3]: Same as https://github.com/Vets-Who-Code/vets-who-code-app/pull/421#discussion_r950646577

File: src/components/ResumeBuilder/resume-builder.js (hunk 7 above)
Comment (andrew-t-james) [reply src/components/ResumeBuilder/resume-builder.js:6]: Same as https://github.com/Vets-Who-Code/vets-who-code-app/pull/421#discussion_r950646577

**Review by Cameron-Porter — APPROVED** [review]
Looks good, just need to resolve the current conflict and should be good to go.

File: src/pages/confirmation.js [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+            <h3>
+              Now that you have an awesome resume, time to go get that dream job!
+            </h3>
+            <div className="rc-response__action">
+              <a href="/resume-builder">
+                  <button className="btn btn-cr">
+                  Create New Resume
+                  </button>
+              </a>
+              <a href="/" id="rtrn-vwc">
```
Comment (andrew-t-james) [inline src/pages/confirmation.js:30]: Please use the Link component for the a element.

File: src/pages/resume-builder.js [hunk 9]
```diff
@@ -18,13 +22,13 @@ const Landing = () => {
...
           <p>
-            Custom-tailor resumes for any job within minutes! Build an easily custimizable and
+            Custom-tailor resumes for any job within minutes! Build an easily customizable and
             simple resume in a few clicks!
           </p>
           <div className="landing-btn">
-            <button type="button" onClick={() => router.push('./ResumeInput.js')}>
-              Next
-            </button>
+              <a href="/resume-input">
```
Comment (andrew-t-james) [inline src/pages/resume-builder.js:29]: +1 Link component

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

**Pull request merged**